core/block/source/source.go

```diff
@@ -27,10 +27,13 @@ import (
 	"github.com/anyproto/anytype-heart/util/slice"
 )
 
+const changeSizeLimit = 10 * 1024 * 1024
```

Comment:
10 mb? isn't it too loose constraint?

Reply:
This question is open for discussion. Value of 10 Mb is stated in the ticket, and moreover @cheggaaa said we can pass up to 256 Mb as a change. Changes with higher sizes will be facing issues while networking